test/e2e/assertion.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"reflect"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/k14s/imgpkg/pkg/imgpkg/imagetar"
+	"github.com/k14s/imgpkg/pkg/imgpkg/lockconfig"
+)
+
+type assertion struct {
+	t *testing.T
+}
+
+func (a *assertion) isBundleLockFile(path, bundleImgRef string) {
```

Comment:
this seems to be main guts of the TestBundleLockFile test. afaik isBundleLockFile is only used in that test. i would recommend inlining it to keep "primary test" directly within Test function.